src/java.base/share/classes/java/text/Format.java

```diff
@@ -160,7 +161,11 @@ protected Format() {
      *            object
      */
     public final String format (Object obj) {
-        return format(obj, new StringBuffer(), new FieldPosition(0)).toString();
+        if ("java.text".equals(getClass().getPackageName())) {
```

Comment:
We can use `==` for performance as getPackageName is interned.

Reply:
Using equals() is fine and the performance is the same.
Using "==" may be fewer characters but carries with it the need to understand that the arguments are interned and that's not always obvious or known.